IntelliTest.NUnit/NUnit2TestFramework.cs

```diff
@@ -376,19 +376,12 @@ protected override bool TryGetCategories( ICustomAttributeProviderEx element, ou
         /// Gets the type of the assertion exception.
         /// </summary>
         /// <value>The type of the assertion exception.</value>
-        public override TypeName AssertionExceptionType
-        {
-            get
-            {
-                System.Diagnostics.Debugger.Launch();
-                return assertionExceptionType ?? (assertionExceptionType = TypeDefinitionName.FromName(
-                    NUnitTestFrameworkMetadata.AssemblyName,
-                    -1,
-                    false,
-                    NUnitTestFrameworkMetadata.RootNamespace,
-                    "AssertionException").SelfInstantiation);
-            }
-        }
+        public override TypeName AssertionExceptionType => assertionExceptionType ?? (assertionExceptionType = TypeDefinitionName.FromName(
```

Comment:
I assume we are going with C# 6 in this project because it targets VS 2015, so we have to build with 2015?